.github/workflows/unit-tests.yml

```diff
@@ -29,15 +29,7 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v2
 
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
-
-      - name: Rust setup
-        run: |
-          bash ./scripts/setup/dev_setup.sh
-
-      - name: Cache Rust
-        uses: Swatinem/rust-cache@v1
```

Comment:
At least the cache for unit tests should be placed separately.

Reply:
thank you for reminding, i'd try another way allowing different cache strategy